### PR TITLE
docs(changelog): initial changelog

### PR DESCRIPTION
**Pull request by rcoringrato-sfdc**
### What does this PR do?

Updates changelog for 50.1.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 50.1.0 - October 14, 2020
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Update LWC preview command to launch a native app or mobile browser ([PR #2585](https://github.com/forcedotcom/salesforcedx-vscode/pull/2585))
+
+- Run SOQL Query Builder and view results ([PR #2565](https://github.com/forcedotcom/salesforcedx-vscode/pull/2565))
```
Comment (ekapner) [inline packages/salesforcedx-vscode/CHANGELOG.md:9]: I thought we weren't mentioning SOQL Builder until it betas on Nov 9. Yes?

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+# 50.1.0 - October 14, 2020
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Update LWC preview command to launch a native app or mobile browser ([PR #2585](https://github.com/forcedotcom/salesforcedx-vscode/pull/2585))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: This change was released last week

Comment (ekapner) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: should we show real name of command? SFDX: Preview Component Locally ?? Is this the one? I vote for real name

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
+# 50.1.0 - October 14, 2020
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:1]: @rcoringrato-sfdc these are the items I think should be part of this week's changelog:

https://github.com/forcedotcom/salesforcedx-vscode/pull/2619
https://github.com/forcedotcom/salesforcedx-vscode/pull/2607
https://github.com/forcedotcom/salesforcedx-vscode/pull/2591
https://github.com/forcedotcom/salesforcedx-vscode/pull/2573

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 2 above)
Comment (ekapner) [reply packages/salesforcedx-vscode/CHANGELOG.md:7]: Yes, I was having a feeling of deja vu.

**Review by ekapner — APPROVED** [review]

**Review by lcampos — APPROVED** [review]

**Pull request merged**